operator/v1/types_clustermanager.go

```diff
@@ -70,6 +70,10 @@ type ClusterManagerSpec struct {
 	// AddOnManagerConfiguration contains the configuration of addon manager
 	// +optional
 	AddOnManagerConfiguration *AddOnManagerConfiguration `json:"addOnManagerConfiguration,omitempty"`
+
+	// ResourceRequirement specify QoS classes of deployments managed by clustermanager
+	// +optional
+	ResourceRequirement *ResourceRequirement `json:"resourceRequirement,omitempty"`
```

Comment:
it should be noted that it applies to all containers since there are multiple containers running in the cluster manager.

Reply:
comment added